modules/fakemeta/engfunc.cpp

```diff
@@ -12,9 +12,12 @@
 //
 
 #include "fakemeta_amxx.h"
+#include <engine_strucs.h>
 
 TraceResult g_tr;
 
+char LightStyleBuffers[MAX_LIGHTSTYLES];
```

Comment:
I meant an array of `AString`, not `char`. `MAX_LIGHTSTYLES` means lightstyle count, not lightstyle max size like `MAX_LIGHTSTYLE_SIZE`.